packages/worker/src/worker-options.ts

```diff
@@ -217,16 +217,18 @@ export interface WorkerOptions {
    * to complete processing accepted Workflow Tasks fast enough. Increasing the number of Workflow threads
    * (see {@link WorkerOptions.workflowThreadPoolSize}) may help in that case.
    *
-   * As a general guidelines:
+   * General guidelines:
    * - High latency to Temporal Server => Increase this number
    * - Very short Workflow Tasks (no lengthy Local Activities) => increase this number
    * - Very long/heavy Workflow Histories => decrease this number
    * - Low CPU usage despite backlog of Workflow Tasks => increase this number
    * - High number of Workflow Task timeouts => decrease this number
    *
-   * In our reference performance test against Temporal Cloud, running with a single Workflow thread and the Reuse V8
-   * Context option enabled, we reached peak performance with a `maxConcurrentWorkflowTaskExecutions` of `120`, and
-   * `maxConcurrentWorkflowTaskPolls` of `60`. Your millage may vary.
+   * In some performance test against Temporal Cloud, running with a single Workflow thread and the Reuse V8 Context
+   * option enabled, we reached peak performance with a `maxConcurrentWorkflowTaskExecutions` of `120`, and
+   * `maxConcurrentWorkflowTaskPolls` of `60` (worker machine: Apple M2 Max; ping of 74 ms to Temporal Cloud;
```

Comment:
I'm wondering what the CPU load was and number of wf/s (not that we need to document that).